apps/meteor/tests/e2e/video-conference.spec.ts

```diff
@@ -35,6 +35,24 @@ test.describe('video conference', () => {
 		await expect(poHomeChannel.content.videoConfMessageBlock.last()).toBeVisible();
 	});
 
+	test.describe('test video conference message block', async () => {
+		test.use({ storageState: Users.admin.state });
+
+		test.beforeAll(async ({ api }) => {
+			await setUserPreferences(api, { displayAvatars: false });
+		});
+
+		test.afterAll(async ({ api }) => {
+			await setUserPreferences(api, { displayAvatars: true });
+		});
+
+		test('should not render avatars in video conference message block', async () => {
```

Comment:
Does it make sense to have different combinations like: should render the avatars if the user preference is true, also, what should happen when the list contains user with and without avatars?

Reply:
I don't see why making a test to check the default assertion for this preference, we already have a test to check if the message block is there and the preference is true by default, so I added a test just to check if its ignoring the avatars properly when changing the preference. 

> what should happen when the list contains user with and without avatars
Basically what I'm testing, if the avatars are there or not

Reply:
There's never "users without avatar" afaik. If the user has no avatar it's the default one...